### DISCUSSION TITLE
Raspberry - how to add another target platforms

### DISCUSSION THREAD

**Discussion by krishtekpl** [docker/buildx#951] [Q&A]
Hello,
I am trying to build an image for linux/amd64 on Rasbbery linux/arm/v7. Unfortunately I can't :(
When I check `docker buildx ls`, I see:
```
NAME/NODE    DRIVER/ENDPOINT             STATUS  PLATFORMS
mybuilder *  docker-container
  mybuilder0 unix:///var/run/docker.sock running linux/arm/v7, linux/arm/v6
default      docker
  default    default                     running linux/arm/v7, linux/arm/v6
```
I went through all the steps explained here: [https://docs.docker.com/desktop/multi-arch/](https://docs.docker.com/desktop/multi-arch/). No Idea what to do more :(

Each help is much appreciated

**Comment by tonistiigi** [comment]
`docker run -it --rm --privileged tonistiigi/binfmt --install amd64` https://github.com/tonistiigi/binfmt#installing-emulators
